public/locales/en/translations.json

```diff
@@ -43,7 +43,10 @@
         "currentWindow": "Currently",
         "previousWindow": "Previously"
       },
-      "map": { "header": "Device locations" },
+      "map": {
+        "header": "Device locations",
+        "runtimeError": "An error occurred while rendering the map"
```

Comment:
Since this is a sentence, shouldn't it end in a period?

Reply:
No, this isn't official copy, just placeholder.